content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,142 @@
+# OpenTelemetry Environment Variable Specification
+
+**Status**: [Mixed](document-status.md)
+
+The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they SHOULD use the names listed in this document.
```

Comment:
Unless there's no impact to using different variable names, it seems that this would be a 'must' vs a 'should':
"If they do, they MUST use the variable names listed in this document" ?

Reply:
I believe 'should' refers to using these variable names as best practices, but could be done with others as well?